src/app/service/guest-pass/AddGuestPass.ts

```diff
@@ -17,17 +17,17 @@ export default async (guestUser: GuildMember): Promise<any> => {
 	notifyUserOfGuestExpiration(guestUser);
 	removeGuestRoleOnExpiration(guestUser);
 	return guestUser.send({
-		content: `You have been granted guest access at Bankless DAO! Guest passes last for 
-		${process.env.DAO_GUEST_PASS_EXPIRATION_DAYS} days and will expire afterwards. To renew your guest pass, ask any 
-		Level 2 contributor to renew it, or post in #get-involved. In the future, we'll automate guest pass renewal based on 
-		your activity in the DAO!` })
-		.catch(e => LogUtils.logError('failed to send message to new guest', e));
+		content: 'You have been granted guest access at Bankless DAO! Guest passes last for '
```

Comment:
`Well look at you! Welcome to BanklessDAO! You've been given a guest pass by ${user}! Go give them a thank you! Your guest pass will last for ${time} at which time you should return to ${user} to renew your guest pass.`

Reply:
Ah yes the text here was requested from frog monkey, should we let him know we're replacing it?

Reply:
Hmm it doesn't seem discord captures who changes the role change when it happens.

Reply:
Oh that’s a bummer. Aight. To be continued.